public-ui/src/app/views/survey-view/survey-view.component.html

```diff
@@ -141,9 +141,17 @@ <h5 class="box-title">
         <div class="row  answers" [hidden]="!showAnswer[q.conceptId]">
           <div class="col-md-7 col-sm-5 col-xs-12 answer-list">
             <div  class="row answer-text"  [class.answer-active]="q.selectedAnswer && q.selectedAnswer.id === a.id" (click)="showAnswerGraphs(q, a)" *ngFor="let a of q.countAnalysis.results">
-              <div class="col-sm-6 col-xs-12">
+              <div class="col-sm-6 col-xs-12" *ngIf="binnedSurveyQuestions.indexOf(a.stratum2) === -1; else binnedSurveyQuestionDisplay">
                 <label class="answer-text" innerHtml="{{a.stratum4 | highlightSearch : searchText.value }} -- {{a.stratum3}}"> {{a.stratum4}} </label>
               </div>
+              <ng-template #binnedSurveyQuestionDisplay>
+                <div class="col-sm-6 col-xs-12" *ngIf="binnedSurveyQuestions.indexOf(a.stratum2) > -1">
```

Comment:
nit: is this needed since we're in the `else` clause already?

Reply:
Missed to remove this.